packages/gatsby/package.json

```diff
@@ -71,6 +71,7 @@
     "gatsby-react-router-scroll": "^2.0.7",
     "gatsby-telemetry": "^1.0.7",
     "glob": "^7.1.1",
+    "got": "^9.6.0",
```

Comment:
We can't use `got@^9` because it is Node 8+ and we still support Node 6 (for a little while at least) :(

Reply:
Great, thanks for the heads-up! I made a quick commit to downgrade.